packages/usa-icon/src/usa-icon.twig

```diff
@@ -42,7 +42,6 @@ function copyHTML(el){
 
   message = document.createElement("span");
   message.setAttribute("id", "icon-copy-notice");
-  message.setAttribute("aria-live", "assertive");
```

Comment:
The current setup does not appear to read out the "Copied to clipboard" message in VoiceOver. Removing this redundant assertive attribute resulted in successful readouts of "Copied to clipboard". 

Note: at times I need to restart the browser and VoiceOver to get consistent "Copied to clipboard" readout.